usersession/userd/launcher.go

```diff
@@ -104,7 +104,7 @@ const launcherIntrospectionXML = `
 //   - https://github.com/snapcore/snapd/pull/8304
 
 var (
-	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg", "slack", "msteams"}
+	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg", "zoomus", "zoomphonecall", "slack", "msteams"}
```

Comment:
I'm a bit confused looking at the documentation. Isn't `zoomus://` only used on ios/android?

Reply:
It's provided by mobile clients but you can paste the URL and open it on a computer. I really don't understand why they have two URL schemes for the same thing.